index.html

```diff
@@ -443,18 +466,22 @@ <h2><dfn>observe()</dfn> method</h2>
             as its only item.</li>
             <li>If <var>options</var>'s <a>buffered</a> flag is set:
               <ol>
-                <li>Let <var>entries</var> be the <a>PerformanceEntryList</a>
-                object returned by <a href="#filter-buffer-by-name-and-type"></a>
-                algorithm with <var>buffer</var> set to <a>relevant
-                performance entry buffer</a>, <var>name</var> set to
-                <code>null</code> and <var>type</var> set to
-                <var>options</var>'s <a>type</a>.
-                </li>
-                <li>Append <var>entries</var> to the <a data-cite=
-                "WHATWG-DOM#context-object">context object</a>'s <a>observer
-                buffer</a>.
-                </li>
-              </ol>
+                <li>Let <var>map</var> be the <a>performance entry buffer map</a>
```

Comment:
Here as well, you can call the "get the relevant tuple" algorithm defined above instead if accessing the map directly

Reply:
Done.